static/bidder-info/adtarget.yaml

```diff
@@ -0,0 +1,10 @@
+maintainer:
+  email: "kamil@adtarget.com.tr"
+capabilities:
+  app:
+    mediaTypes:
+      - banner
```

Comment:
Should the adapter code enforce that video impressions aren't valid for app requests?

Reply:
hm, actually there is no reason to ignore video for app